doc/api_reference.md

```diff
@@ -349,6 +350,28 @@ cancellation of the operation returned by the function. Note that cancellation
 may also be requested through the stop-token of the receiver that is connected
 to the sender returned by `let_value_with_stop_source()`.
 
+### `let_value_with_stop_token(Invocable func) -> Sender`
+
+The `let_value_with_stop_token()` algorithm creates an
+`inplace_stop_token_adapter` that remains alive for the duration of an
+operation.
+
+`func` is invoked with an lvalue reference to an `inplace_stop_token`
+created by subscribing the `inplace_stop_token_adapter` to the stop token
+returned by the invocation of `get_stop_token` on the provided receiver.
+This invocation must return a Sender.
+
+The reference passed to `func` will remain valid until the returned sender
+completes, at which point the `inplace_stop_token_adapter` goes out of scope.
+
+For example:
+```c++
+let_value_with_stop_token(
```

Comment:
I wonder if this should be named more simply `let_stop_token` or `let_current_stop_token` instead of `let_value_with_stop_token`.

Reply:
I'm not too familiar with the naming of the other let_* operators, but I used the same naming scheme as the `let_value_with_stop_source` -- I think it'd be okay to rename this operator but then it would inconsistent with the stop source variant.